### PR TITLE
HTMLElement.prototype.draggable

### LINKED ISSUE #2268: `HTMLElement.prototype.draggable` descriptor is missing
### Basic info:

- **Node.js version:** v10.1.0
- **jsdom version:** 11.11.0

### Minimal reproduction case

```js
const jsdom = require("./");
const { JSDOM } = jsdom;
const dom = new JSDOM(`<body></body>`);

const HTMLElement = dom.window.HTMLElement;
const draggable = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'draggable');

console.log(draggable); // yields undefined
```

Documentation:
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable

### How does similar code behave in browsers?

```js
Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'draggable');
```

It yields `{get: ƒ, set: ƒ, enumerable: true, configurable: true}` in all major browsers.

### PR DESCRIPTION
**Pull request by pmdartus**
This PR adds support for the [draggable](https://html.spec.whatwg.org/multipage/dnd.html#the-draggable-attribute) global HTML attribute.

I also have been investigating adding support for the rest of the [Drag and Drop](https://html.spec.whatwg.org/multipage/dnd.html#dnd) APIs, but I am not sure it would make sense to add support for those in the context of jsdom.

Fix #2268

### CODE REVIEW

**Comment by diervo** [comment]
@domenic The first of many :)

File: lib/jsdom/living/nodes/HTMLAnchorElement-impl.js [hunk 1]
```diff
@@ -16,6 +16,22 @@ class HTMLAnchorElementImpl extends HTMLElementImpl {
   set text(v) {
     this.textContent = v;
   }
+
+  get draggable() {
```
Comment (domenic) [inline lib/jsdom/living/nodes/HTMLAnchorElement-impl.js]: This shouldn't be here; it's an attribute for HTMLElement, not for HTMLAnchorElement. So only implement it on HTMLElement.

File: lib/jsdom/living/nodes/HTMLElement-impl.js [hunk 2]
```diff
@@ -75,6 +75,22 @@ class HTMLElementImpl extends ElementImpl {
     this._clickInProgress = false;
   }
 
+  get draggable() {
```
Comment (domenic) [inline lib/jsdom/living/nodes/HTMLElement-impl.js]: Implement all the logic here. Test if `this._localName === "img"` or `this._localName === "a"`.

File: lib/jsdom/living/nodes/HTMLElement-impl.js [hunk 3]
```diff
@@ -75,6 +75,22 @@ class HTMLElementImpl extends ElementImpl {
...
+    const attributeValue = this.getAttribute("draggable");
+    return attributeValue === "true";
+  }
+  set draggable(value) {
```
Comment (domenic) [inline lib/jsdom/living/nodes/HTMLElement-impl.js]: Just do `this.setAttribute("draggable", String(value))`.

**Comment by pmdartus** [comment]
Thanks for the review @domenic, I updated PR based on your feedbacks.

**Comment by domenic** [comment]
Thanks so much for this, release coming soon! Also I wanted to say I appreciate a lot that you immediately did the right thing with the web platform tests framework; many first-time contributors miss that. Looking forward to more!

**Pull request merged**